WcaOnRails/app/views/competitions/_nearby_competitions.html.erb

```diff
@@ -54,8 +54,35 @@
       Last <%= Competition::NEARBY_DAYS_INFO %> days
     </label>
     <div class="col-sm-9">
-      <% nearby_count = @competition.nearby_competitions(Competition::NEARBY_DAYS_INFO, Competition::NEARBY_DISTANCE_KM_DANGER).count %>
-      <div class="<%= nearby_count < Competition::NEARBY_INFO_COUNT ? "alert alert-success" : "alert alert-danger" %>" role="alert"><%= nearby_count %> <%= "competition".pluralize(nearby_count) %> within <%= Competition::NEARBY_DISTANCE_KM_DANGER %>km and <%= Competition::NEARBY_DAYS_INFO %> days.
+      <% nearby_info = @competition.nearby_competitions(Competition::NEARBY_DAYS_INFO, Competition::NEARBY_DISTANCE_KM_DANGER).sort_by { |i| i.start_date } %>
```

Comment:
Does `...sort_by(&:start_date)` work here instead?

I don't like that we're using `NEARBY_DAYS_INFO` alongside `NEARBY_DISTANCE_KM_DANGER`. Can we use `NEARBY_DAYS_INFO` and `NEARBY_DISTANCE_KM_INFO` instead?